profile/README.md

```diff
@@ -0,0 +1,21 @@
+<h1 align="center">Run JavaScript <a href="https://nodejs.org/en/download">Everywhere</a>.®</h1><br>
```

Comment:
Is it necessary to add a trademark ® symbol here?

Reply:
Well, on the website we use the trademark symbol. We can remove it, I'm neutral.

Reply:
You mean on https://nodejs.dev/en/ ? I don't see the symbol:
![CleanShot 2023-03-22 at 09 05 29](https://user-images.githubusercontent.com/2352663/226838290-9b121076-70c1-4979-9ac3-adb9ea40739f.png)

Reply:
Oh, I misread the comment (on phone) yes let me definitely remove the trademark from the h1, I thought @legendecas was talking about the "Get Node.js" link.

I forgot to remove the trademark symbol on the h1 because on my draft the title was something different ending with Node.js